java/provision.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# installation settings
+PROJECT="java_project" # we would want a name passed to it via te first argument, $1
+DB="fcc_java_db" # the name of postgreSQL DB we need to provision, maybe $2
+
+# This file is executed by root user - sudo not needed
+# But do not create any directory
+# which vagrant user might need access to later in su mode
+# use su - vagrant -c "" syntax
+export DEBIAN_FRONTEND=noninteractive
+echo "---------------------------------------------"
+echo "Running vagrant provisioning"
+echo "---------------------------------------------"
+
+# install heroku toolbelt
+echo "-------------- Installing heroku toolbelt -------------------------"
+wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
+# These shell script snippets are directly taken from heroku installation script
+# We want to avoid the apt-get update
+# add heroku repository to apt
+echo "deb http://toolbelt.heroku.com/ubuntu ./" > /etc/apt/sources.list.d/heroku.list
+# install heroku's release key for package verification
+wget -O- https://toolbelt.heroku.com/apt/release.key 2>&1 | apt-key add -
```

Comment:
Pipe output to `/dev/null`. Otherwise it gives text from line [49](https://gist.github.com/alayek/44cf2f4a2747f4684c8a7709f2c3c647#file-logfile-txt-L49) to line [134](https://gist.github.com/alayek/44cf2f4a2747f4684c8a7709f2c3c647#file-logfile-txt-L134) in `stderr` red.

I think I fixed this earlier, but don't know how it vanished from the `python` part of the repo :confused: